src/jdk.management/share/native/libmanagement_ext/DiagnosticCommandImpl.c

```diff
@@ -203,10 +213,12 @@ Java_com_sun_management_internal_DiagnosticCommandImpl_getDiagnosticCommandInfo
           free(dcmd_info_array);
           return NULL;
```

Comment:
Again aren't you missing the PopLocalFrame on this return path?

Reply:
Fixed.